src/gui/MainWindow.cpp

```diff
@@ -1033,8 +1033,8 @@ void MainWindow::updateWindowTitle()
 
     if (stackedWidgetIndex == DatabaseTabScreen && tabWidgetIndex != -1) {
         customWindowTitlePart = m_ui->tabWidget->tabName(tabWidgetIndex);
-        if (isModified) {
-            // remove asterisk '*' from title
+        if (!isModified) {
+            // Remove asterisk '*' from title
             customWindowTitlePart.remove(customWindowTitlePart.size() - 1, 1);
         }
         m_ui->actionDatabaseSave->setEnabled(m_ui->tabWidget->canSave(tabWidgetIndex));
```

Comment:
Where? This might be actually needed because the tab name has been set with the asterisk. If that is not removed, the window title placeholder will show `**`.